examples/common/buildAccountSigner.ts

```diff
@@ -5,44 +5,48 @@ import {
     AccountAddress,
     signMessage,
     buildAccountSigner,
+    parseWallet,
 } from '@concordium/node-sdk';
 
 const cli = meow(
     `
   Usage
-    $ yarn ts-node <path-to-this-file> [options]
+    $ yarn run-example <path-to-this-file> [options]
 
   Required
-    --keyFile,   -f  A file containing the private key(s) of an account, which must be a supported format (e.g. a private key export from a Concordium wallet)
```

Comment:
As you now get the account address from the file, I'd change this to explicitly state that it _only_ supports the wallet export.